imagej/imagej-ops2-tutorial/docs/SearchingForOps.md

```diff
@@ -0,0 +1,12 @@
+# Searching for Ops in the Environment
```

Comment:
Is there a way to generate a tree or list output of all ops that are available to the environment? If so, that would be a nice section to add to this doc.

Reply:
You could use `ops.descriptions()` (no-args ctor) - I *really* don't want to include that printout here, though - it'd be way too long.

I did add a note in e8c84471